### PR TITLE
[14.0][ADD] mrp_unbuild_valuation_layer_link

### PR DESCRIPTION
**Pull request by ThiagoMForgeFlow**
This module shows a valuation button in the unbuild orders that is connected with valuation layers of the order.

@ForgeFlow

### CODE REVIEW

File: mrp_unbuild_valuation_layer_link/tests/test_mrp__unbuild_valuation_layer_link.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                    unbuild.produce_line_ids + unbuild.consume_line_ids
+                ).stock_valuation_layer_ids.ids,
+            )
+        ]
+        self.assertEqual(
+            result["domain"],
+            domain,
+            "You should have as domain the ids of the stock valuation belonging "
+            "to the ids of the stock moves of produce_line and consume_line",
+        )
```
Comment (LoisRForgeFlow) [inline mrp_unbuild_valuation_layer_link/tests/test_mrp__unbuild_valuation_layer_link.py]: you should not test the domain definition, but that this domain returns some layers, and that those layers are the same that you expect (search them by another mean).

**Review by BernatPForgeFlow — APPROVED** [review]
Functional review OK!

**Review by DavidJForgeFlow — APPROVED** [review]
Functional review :+1:

File: mrp_unbuild_valuation_layer_link/tests/test_mrp__unbuild_valuation_layer_link.py (hunk 1 above)
Comment (etobella) [reply mrp_unbuild_valuation_layer_link/tests/test_mrp__unbuild_valuation_layer_link.py]: @ThiagoMForgeFlow Can you check @LoisRForgeFlow comment?

Comment (ThiagoMForgeFlow) [reply mrp_unbuild_valuation_layer_link/tests/test_mrp__unbuild_valuation_layer_link.py]: It's done, thank you

File: mrp_unbuild_valuation_layer_link/tests/test_mrp__unbuild_valuation_layer_link.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            self.env["stock.move"].search([]).stock_valuation_layer_ids.ids
+        )
+
+        result = unbuild.action_view_stock_valuation_layers()
+        domain = result["domain"]
+        unbuild_valuation_layers = domain[0][2].sort()
+        difference_layers = list(
+            set(layers_after_unbuild) - set(layers_before_unbuild)
+        ).sort()
+
```
Comment (LoisRForgeFlow) [inline mrp_unbuild_valuation_layer_link/tests/test_mrp__unbuild_valuation_layer_link.py:46]: Could you also add here:

`self.assertTrue(difference_layers)`

?

Comment (ThiagoMForgeFlow) [reply mrp_unbuild_valuation_layer_link/tests/test_mrp__unbuild_valuation_layer_link.py:46]: Done

**Comment by LoisRForgeFlow** [comment]
@ThiagoMForgeFlow Perfect, just squash both commits together and this will be ready to merge.

**Pull request merged**